src/System.Text.Primitives/System/Text/Extensions.Float.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.Text
+{
+    public static partial class Extensions
+    {
+        public static bool TryFormat(this double value, Span<byte> buffer, out int bytesWritten, ParsedFormat format = default, SymbolTable symbolTable = null)
```

Comment:
Float implies "floating-point". That would encapsulate double, float and possible decimal at some point.